src/components/Bookings/Bookings.jsx

```diff
@@ -1,10 +1,60 @@
 import Search from "@/components/Search/Search";
 import SearchResults from "@/components/SearchResults/SearchResults.jsx";
 import FakeBookings from "@/data/fakeBookings.json";
-import { useState } from "react";
+import { useState, useEffect } from "react";
+import "./Bookings.scss";
 
 const Bookings = () => {
-  const [bookings, setBookings] = useState(FakeBookings);
+  const [bookings, setBookings] = useState([]);
+  const [newBooking, setNewBooking] = useState({
+    firstName: "",
+    surname: "",
+    email: "",
+    title: "",
+    roomId: "",
+    checkInDate: "",
+    checkOutDate: "",
+  });
+  const [isModalOpen, setModalOpen] = useState(false);
+
+  const openModal = () => {
+    setModalOpen(true);
+  };
+
+  const closeModal = () => {
+    setModalOpen(false);
+  };
+
+  const capitalizeFirstChar = (value) => {
+    return value.charAt(0).toUpperCase() + value.slice(1);
+  };
+
+  const handleInputChange = (e, fieldName) => {
+    const updatedValue = capitalizeFirstChar(e.target.value);
+    setNewBooking({ ...newBooking, [fieldName]: updatedValue });
```

Comment:
Why did you add this ` [fieldName]: updatedValue }` ? Is it enabling the user to upload a file?